index.html

```diff
@@ -254,11 +277,21 @@ <h2>
           </li>
         </ul>
       </form>
+       </div>
+       </div>
+    </div>
       <div class="full">
               <img class="image-down"src="img/contact-form.png" alt="bottom image">
+              <img class="image-first"src="img/first.png" alt="bottom image">
+              <img class="image-middle"src="img/middle.png" alt="bottom image">
+              <img class="image-last"src="img/last.png" alt="bottom image">
+      </div>
+      <div class="dash-line">
+             <img class="rectangle"src="img/rectangle.png" alt="bottom image">
```

Comment:
- [ ] Kindly add some space and a divider on top of the social media links to match the design in Figma 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/206476620-74285493-dd06-4a39-94f9-3620647d62c6.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/206476681-0a1f013d-0f00-4bef-bf1b-d64de8148eb4.png)